GLAA.ViewModels/LicenceApplication/OrganisationDetailsViewModel.cs

```diff
@@ -301,21 +318,82 @@ public VATStatusViewModel()
         // https://en.wikipedia.org/wiki/VAT_identification_number
         [RegularExpression(@"[a-zA-Z]{2}[a-zA-Z0-9 ]{2,13}", ErrorMessage = "Please enter a valid VAT registration number")]
         [RequiredIf(ErrorMessage = "The VAT registration number field is required")]
-        [Display(Name = "VAT Number", Description = "For example GB999 9999 73")]
+        [Display(Name = "VAT Registration Number", Description = "For example GB999 9999 73")]
         public string VATNumber { get; set; }
 
         [UIHint("_NullableDateTime")]
-        [Display(Name = "VAT Registration Date", Description = "Please enter the date you registered for VAT.")]
+        [Display(Name = "VAT Registration Date")]
         public DateViewModel VATRegistrationDate { get; set; }
 
         public bool IsRequired => HasVATNumber ?? false;
+
+        public bool IsValid { get; set; }
+
+        public void Validate()
```

Comment:
Could we make a version of `YesNoViewModel` that extends `Validatable`? It would save a lot of this code repetition.

Reply:
That's a great idea. Too many times I've though "damn it already inherits YesNo so I can't inherit Validatable"